python/dgl/graphbolt/feature_fetcher.py

```diff
@@ -215,8 +215,6 @@ def wait(self):
         if self.edge_feature_keys and num_layers > 0:
             for i in range(num_layers):
                 original_edge_ids = data.edge_ids(i)
-                if original_edge_ids is None:
```

Comment:
why is necessary to remove this check?

Reply:
If features are available, feature fetcher silently runs and fetches no features due to this line. This was when return eids is False. Do you think we should remove this line?

Reply:
Yes, I expect it to be never None.

Reply:
And the tests were broken, not testing edge feature fetch at all.